docs/background-color.md

```diff
@@ -41,26 +41,20 @@ Control the background color of an element using the `bg-{transparent|inherit|cu
 ### Hover, focus and other states
 
 Conditionally apply utility classes in different states using variant modifiers.
-For example, use `hover:s-bg-positive` to only apply the `s-bg-positive` utility on hover.
+For example, use `hover:s-bg-hover` to only apply the `s-bg-hover` utility on hover.
 
 <container>
   <div class="grid grid-cols-3 gap-16 justify-items-center">
-    <div class="h-80 w-80 border rounded-16 s-bg hover:s-bg-hover active:s-bg-active active:hover:s-bg-active-hover"></div>
-    <div class="h-80 w-80 border rounded-16 s-bg-info hover:s-bg-info-hover active:s-bg-info-active active:hover:s-bg-info-active-hover"></div>
-    <div class="h-80 w-80 border rounded-16 s-bg-warning hover:s-bg-warning-hover active:s-bg-warning-active active:hover:s-bg-warning-active-hover"></div>
+    <div class="h-80 w-80 border rounded-16 s-bg hover:s-bg-hover active:s-bg-active"></div>
```

Comment:
It's again beyond the scope of this PR, but I'm afraid these interactive elements should not be represented by `div`s, but rather by `button` or `a` elements. The docs need to really be looked into with accessibility in mind 😄

Reply:
Is there a task for it?

Reply:
I think this one would be the one: https://nmp-jira.atlassian.net/browse/WARP-263